src/components/CryptocurrienciesRate/CryptocurrenciesRate.tsx

```diff
@@ -0,0 +1,46 @@
+import ICryptoCurrienciesRateView from "@/constants/interfaces/CryptoCurrenciesRateView.interface";
+import CryptocurrenciesRateRow from "./CryptocurrenciesRateRow";
+
+const CryptocurrenciesRate = ({
+  cryptoCurrenciesRateViews,
+  showCodicon,
+}: {
+  cryptoCurrenciesRateViews: ICryptoCurrienciesRateView[];
+  showCodicon: boolean;
+}) => {
+  const halfData = Math.ceil(cryptoCurrenciesRateViews.length / 2);
+
+  return (
+    <div className="grid grid-cols-2 gap-4 text-dirty-white">
+      <div className="border-r border-primary">
+        {cryptoCurrenciesRateViews
+          .slice(0, halfData)
+          .map((cryptoCurrenciesRateView, index) => (
+            <CryptocurrenciesRateRow
+              key={index}
+              cryptoCurrienciesRateView={cryptoCurrenciesRateView}
+              index={index}
+              showCup={true}
+              showCodicon={showCodicon}
+              className="mr-5"
+            />
```

Comment:
I'd rather have this slice logic used inside of the <CryptocurrenciesRateRow />.
Add a helper.tsx file and store these data processing operations there.

Also it does not look safe with cutting half data. You are assuming you will get 10 objects from backend which might not be the case. This Component is also non responsive. Add styles so it is at least wrapping nicely. Make it less depeneded on the data delivered and just take first 10 object and display them nicely.